cfme/intelligence/reports/reports.py

```diff
@@ -469,6 +469,10 @@ def exists(self):
         except (CandidateNotFound, TimedOutError):
             return False
 
```

Comment:
Lets not add this same method 150 times to all the different entity classes. Just define it in BaseEntity, along with a default definition of `exists` that looks for a `'Details'` navigation destination and goes to it.

Reply:
https://github.com/ManageIQ/integration_tests/pull/8552